package.json

```diff
@@ -88,7 +88,7 @@
     "karma-coverage": "2.1.0",
     "karma-firefox-launcher": "2.1.2",
     "karma-jasmine": "4.0.1",
-    "karma-rollup-preprocessor": "7.0.7",
+    "karma-rollup-preprocessor": "github:jlmakes/karma-rollup-preprocessor#master",
```

Comment:
Let's wait what they say, maybe we can avoid using this and use the next update.

Reply:
:+1: